projects/ui/src/lib/components/po-field/po-lookup/po-lookup.component.spec.ts

```diff
@@ -324,6 +324,24 @@ describe('PoLookupComponent:', () => {
       expect(component.inputEl.nativeElement.value).toBe('123 - teste');
     });
 
+    it('setInputValueWipoieldFormat: should define the input value and the old value with the value returned from formatFields with the fieldFormat being an array', () => {
+      component.fieldFormat = ['label', 'value'];
+      component['setInputValueWipoieldFormat'](objectSelected);
+      expect(component['oldValue']).toBe('teste - 123');
+      expect(component.inputEl.nativeElement.value).toBe('teste - 123');
+    });
+
+    it('formatFields: should formats the return based on the array sent by fieldFormat ', () => {
+      component.fieldFormat = ['label', 'value'];
+      expect(component['formatFields'](objectSelected, component.fieldFormat)).toBe('teste - 123');
+    });
+
+    it('should format the return based on the default fieldValue sent if the fieldFormat is undefined', () => {
+      component.fieldFormat = ['label', 'value'];
+      component.fieldValue = 'value';
+      expect(component['formatFields'](objectSelected, undefined)).toBe(123);
+    });
+
```

Comment:
`` `

it('formatFields: ', () => {
     // CNPJ não contém em objectSelected
      component.fieldFormat = ['label', 'cnpj'];

      expect(component['formatFields'](objectSelected, component.fieldFormat)).toBe('teste');
});

it('formatFields: ', () => {
      component.fieldValue = 'value';
      component.fieldFormat = 'abc'; // valor invalido

      expect(component['formatFields'](objectSelected, component.fieldFormat)).toBe(123);
});
```